internals/getCsp.js

```diff
@@ -0,0 +1,29 @@
+const csp = `
+    default-src 'none';
+    script-src
+      'self'
+      'report-sample';
+    style-src
+      'self'
+      fonts.googleapis.com
```

Comment:
This should go away once https://github.com/oasisprotocol/oasis-wallet-web/issues/526 is implemented.